### ISSUE TITLE
Canadian mysterious codes

### ISSUE THREAD

**Issue by jpmckinney** [lynnandtonic/airport-codes#799]
Several Canadian codes are describe as being "one of the mysterious codes in which the other two letters were randomly assigned". Like YUL (#54), most of these have a radio beacon that broadcasts the two last letters including:

* LT
* MM
* QB
* QT
* WG
* QY
* XH
* XX
* XY
* YT
* ZF

Source: http://www.worldaerodata.com/nav/Canada.php

**Comment by lynnandtonic** [comment]
Makes sense! I'm going to do some more research regarding which came first.
